### ISSUE TITLE
27.0.0 - cell content moved 9 px down?

### ISSUE THREAD

**Issue by spyro2000** [ag-grid/ag-grid#4912]
**I'm submitting a ...**  (check one with "x")
```
[] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[x] support request => see 'Requesting Community Support'
```

**26.0.2 Behavior**

With ag-grid 26.2.0, my cells with buttons were correctly rendered like this:

![image](https://user-images.githubusercontent.com/8479925/153247706-26290bda-d841-40cf-8820-6c6fcc231dc3.png)

**27.0.0 Behavior**

Since 27.0.0, there is a strange offset about 9px from the top, with the overflow hidden (ignore the different numbers):

![image](https://user-images.githubusercontent.com/8479925/153247910-7e10bab8-9244-41d6-ab34-c9d3f2cb6f4f.png)

I tried to fix this manually by CSS, but wasn't successful. It seems like something changed in the definition of the style _ag-cell_ forcing an additional padding top/bottom , which I am not able to override to simulate the behavior of 26.0.2.

Basically, I want to render the cells with buttons in it without any padding so they are not cut-off.

**Please tell us about your environment:**
Win10 Prof, Angular 13

* **AG Grid version:**  
27.0.0

* **Browser:**
Chrome  97.0.4692.99 64 Bit

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
Typescript/CSS

**Comment by kiril-matev** [comment]
Hi @spyro2000,

Thank you for reporting this.

Can you provide a sample reproducing this issue? 

We need this so we can test your specific scenario to make sure that's covered by any potential fix.

**Comment by rmc-software** [comment]
This has been patched in v27.0.1

**Comment by spyro2000** [comment]
Perfect, thank you! :)

**Comment by spyro2000** [comment]
Seems ag-grid-angular is still on 27.0.0...?

**Comment by StephenCooper** [comment]
Hi @spyro2000 I am trying to recreate the issue but have not able to when using ag-grid-community 27.0.1.

I have created these two plunkers where the only difference is the following line. 

https://plnkr.co/edit/5YirO9Ta2OizM0af
```ts
var systemJsMap = {                
                "ag-grid-community": "https://unpkg.com/ag-grid-community@27.0.1/",
                "ag-grid-angular": "https://unpkg.com/ag-grid-angular@27.0.0/"
            };
```

https://plnkr.co/edit/GE1km1rlGfX7FwVq
```ts
            var systemJsMap = {                
                "ag-grid-community": "https://unpkg.com/ag-grid-community@27.0.0/",
                "ag-grid-angular": "https://unpkg.com/ag-grid-angular@27.0.0/"
            };
```

Flicking between the two seems to show that the padding has been resolved so I am not sure why this isn't the case for you. 

The angular package was not updated as this is just a angular wrapper that will not impact the cell styling.
